content/en/seachat/seachat-manual/06-live-agent-transfer.md

```diff
@@ -0,0 +1,67 @@
+
+At the bottom of the screen, you can see the checkbox that says ***Users can request a live agent during chat***. Make sure to check this box to enable the live agent feature.
+
+> :exclamation: **IMPORTANT** :exclamation:
+> 
+> If you chose to enable the feature of the live agent, your customer will now have the option to request a live agent during the chat conversation. This requires you to provide a real-human agent that will monitor and response to the user.
+
+<br/>
+<a href="/images/product-updates/seachat/en/live-agent-transfer/20240325-live-agent-transfer-1.png" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/live-agent-transfer/20240325-live-agent-transfer-1.png" alt="">
+</a>
+<br/>
+
+## Conversations
+Find the **Conversations** section from the sidebar menu on the left. Here you can see all the conversations that your AI agent has had with your customers. You can see the conversation history and the status of the conversation and this is the control center of your live agent's conversation.
+
+If a customer requests a live agent during the chat conversation, the human agent will see the notification pops up to indicate that the request. If the list of conversation gets too long, simply click on the **Live Agent** button to see the list of conversations that live agents need to complete.
+
+<br/>
+<a href="/images/product-updates/seachat/en/live-agent-transfer/20240325-live-agent-transfer-2.png" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/live-agent-transfer/20240325-live-agent-transfer-2.png" alt="">
+</a>
+<br/>
+<br/>
+
+That's it. The transfer to a live agent is now set up. You can have a live agent take over the conversation to further assist your customer. Once the live agent has completed the request that the customer has, the live agent must click on the **Complete** button to give the conversation back to the AI agent.
+
+<br/>
+<a href="/images/product-updates/seachat/en/live-agent-transfer/20240325-live-agent-transfer-3.png" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/live-agent-transfer/20240325-live-agent-transfer-3.png" alt="">
+</a>
+<br/>
+
+Once the live agent has completed the conversation, the conversation will be marked as completed and the customer will be notified that the conversation has been completed. Live agent can also reactivate the conversation when needed. Simply click on the **Reactivate** button to restart the conversation.
+
+
```

Comment:
Add sections: 
- How to Request Live Agent on Webchat
Add notes that the Live Agent Transfer works with Shopify and Squarespace. They are essentially webchat
- How to Request Live Agent on LINE
![Jun Mao Image Mar 26](https://github.com/seasalt-ai/wiki/assets/20276372/05c8f641-52c4-4b9e-83ed-221c6ba020dc)

- How to Request Live Agent on WhatsApp
@xuchen screenshot?
- How to Request Live Agent on Messenger
@xuchen screenshot?

Reply:
we will do WhatsApp and Messenger on separate PRs